www/templates/intro/intro.html

```diff
@@ -3,6 +3,9 @@
       <ion-slide>
         <ng-include src="'templates/intro/summary.html'"></ng-include>
       </ion-slide>
+      <ion-slide>
+        <ng-include src="'templates/intro/study-text.html'"></ng-include>
```

Comment:
should it go before or after the summary?

Reply:
I feel like before the summary is better, because other studies may want to have other summaries.
You could change the summary too, to say something like "TripAware is a study ...."
Nobody said that the summary has to be the e-mission summary.